test/config/utility.cc

```diff
@@ -313,16 +313,23 @@ std::string ConfigHelper::discoveredClustersBootstrap(const std::string& api_typ
 }
 
 // TODO(#6327) cleaner approach to testing with static config.
-std::string ConfigHelper::adsBootstrap(const std::string& api_type) {
-  return fmt::format(
-      R"EOF(
+// TODO(fredlas) set_node_on_first_message_only was true; the delta+SotW unification
+//               work restores it here. (moved from ads_integration.h)
+std::string ConfigHelper::adsBootstrap(const std::string& api_type,
+                                       envoy::config::core::v3::ApiVersion api_version) {
+  // Note: do not use CONSTRUCT_ON_FIRST_USE here!
```

Comment:
Probably a redundant comment that can go.

Reply:
done